package.json

```diff
@@ -95,6 +95,7 @@
         "@vue/test-utils": "^2.2.6",
         "autoprefixer": "^10.4.13",
         "babel-loader": "^9.1.0",
+        "base64url": "^3.0.1",
```

Comment:
Rather than introduce another dependency, I believe this is covered by `base64URLEncode` in [forge](https://github.com/flowforge/flowforge/tree/9d4925b4a12e0de021b54b595ffae62ef92fb94f/forge)/[db](https://github.com/flowforge/flowforge/tree/9d4925b4a12e0de021b54b595ffae62ef92fb94f/forge/db)
/utils.js